lib/acts_as_taggable_on_mongoid/models/concerns/tag_associations.rb

```diff
@@ -10,6 +10,18 @@ module TagAssociations
           ### ASSOCIATIONS:
 
           has_many :taggings, dependent: :destroy, class_name: "ActsAsTaggableOnMongoid::Models::Tagging"
+          belongs_to :tagger, polymorphic: true, optional: true, index: true
+
+          after_save :atom_unset_blank_tagger
+        end
+
+        private
+
+        def atom_unset_blank_tagger
+          return if !attributes.key?("tagger_id") && !attributes.key?("tagger_type")
+          return if tagger_id.present? && tagger_type.present?
+
+          unset(:tagger_id, :tagger_type)
```

Comment:
Added this to prevent values that are set to nil in the database.  It just makes things simpler.